src/layer/oneway.js

```diff
@@ -79,6 +67,8 @@ export const surface = {
     ],
     "symbol-placement": "line",
     "icon-rotation-alignment": "map",
+    // Assumes driving on the right.
+    "icon-offset": [0, 10],
```

Comment:
I like the offset effect, but it looks really clunky in left-hand-drive localities, including the US Virgin Islands. Perhaps an asymmetric oneway icon could make it look a little better.

Reply:
I was thinking this might fall under the notion that we’re primarily optimizing for North America, but a regression in left-hand driving countries would be far from ideal. We could bundle a GeoJSON of left-hand driving countries and reference it within a `within` expression, similar to #749. Performance might be an issue, however.

Reply:
Really the effect I’m going for is how printed maps traditionally would append an arrow to the (offset) label of each one-way street, rather than treating it as a separately placed repeating element. That would have the benefit of less repetition. But I don’t think that’s feasible here, because the `oneway` field is only present on the `transportation` layer.

Reply:
I guess [25.7&nbsp;kilobytes](https://github.com/openstreetmap/iD/blob/3f35a35dd14dfed6b4dbe62eee64e1c974247332^/data/drive-left.json) or [57.2&nbsp;kB](https://github.com/Project-OSRM/osrm-backend/blob/367933fc1a9705e8ea852b8470f9db2d0f6f73fb/data/driving_side.geojson) isn’t too bad, though it sure would be nice if every road just had its country code on it.